datetime_tz/__init__.py

```diff
@@ -123,12 +150,92 @@ def localtz_set(timezone):
   global _localtz
   _localtz = _tzinfome(timezone)
 
+def require_timezone(zone):
+  """Raises an AssertionError if we are not in the correct timezone"""
+  assert localtz().zone == zone,\
+      "Please set your local timezone to %(zone)s (either in the machine,\
+      or on Linux by exporting TZ=%(zone)s" % {"zone": zone}
+
+# The following code is a workaround to
+# GetDynamicTimeZoneInformation not being present in win32timezone
+
+class SYSTEMTIME_c(ctypes.Structure):
```

Comment:
We can keep require_timezone here.

Let's move the ctypes stuff+_detect_timezone_windows into a file we only import on windows.